src/main/java/no/sikt/sws/WorkspaceStripper.java

```diff
@@ -15,6 +15,7 @@
 public class WorkspaceStripper {
 
     private static final Logger logger = LoggerFactory.getLogger(WorkspaceStripper.class);
+    public static final String INDEX = "_index";
```

Comment:
We should find a more readable name for this. What is it used for? Its clearly not a index
Is it the name of the index-field in json? Maybe something like INDEX_FIELD_NAME would be better?